revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+In case of other detrimental events, Beamswap would also halt rewards and return the remaining grant amount to the Moonbeam Treasury.
+
+Expected TVL is an estimate based on the current token prices and network activities.
+
+### Phase 1
+
+In the first four weeks, a liquidity-sufficient order book would be developed for assets included in the Beamex Perpetual Index token, BLP. With the launch of Beamswap v3 concentrated liquidity AMM and proposed TVL, ~95% of all Moonbeam-based transactions to date can be covered.
+
+Through the use of XCM transfer and connected contracts, a stable pool (3pool) consisting of USDTxc, USDCaxl, and USDCmulti would be deployed.
+
+![|602x339](upload://stGnLWpgEEQMwhCLHXfGMDckMSP.jpeg)
+
+Achieving the forecasted TVL would lead to more daily volume on the Beamswap DEX while increasing the Moonbeam Network key metrics in active users, transactions, and total value locked.
+
+Expected APY and TVL should be met within the first week of the Beamswap v3 launch.
+
+The success of Phase 1 and Beamswap’s capacity to proceed with Phase 2 would be measured by unique user growth, transaction size growth, TVL, interactions with the exchange’s smart contracts, and more.
+
+### Phase 2
+
+Following the successful Phase 1 of building sufficient liquidity, the Beamex perpetual DEX, the first perpetual exchange on Polkadot Network, is planned to launch.
+
+Using Beamswap Flywheel, users will be able to benefit from Moonbeam and Polkadot capabilities to open long or short positions on Beamex-supported assets.
+
+![|602x339](upload://4UU32xt3zIprYMzEIq4fgiebraE.jpeg)
+
+Expected APY and TVL should be met within the second week of the Beamex launch. The adoption of Beamex will be supported through a set of guides, dedicated community managers, and daily marketing activity to ease user onboarding to a completely new product. The use of Beamex Beta is strongly encouraged.
+
+## Roll out of BLP token powering Beamex liquidity
+
+### BLP token
+
+BLP, Beamex Liquidity Provider Index Token, is Beamex’s native liquidity provider asset.
+
+It represents an index of assets used to support swaps and leverage trading on Beamex, and replaces the liquidity token pairs used to provide liquidity on Beamswap.
+
+BLP is composed of:
+
+* ETH (30%)
+* BTC (12%)
+* DOT (5%)
+* GLMR (3%)
+* USDC (50%)
+
+BLP can be minted using any index asset or burned to redeem any index asset. In case the situation with USDC and Circle deteriorates, the stablecoin would be replaced with USDT (Tether) prior to product launch.
+
+Leverage traders source liquidity by borrowing from exchange, or, more accurately, liquidity providers or BLP token holders. When such traders suffer a loss, BLP holders earn a profit and vice versa.
+
+### Phase 1
+
+The initial phase is the Launch phase with 140,000 GLMR community rewards.
+
+### Phase 2
+
+The following phase is a Retail phase with 310,000 GLMR community rewards.
+
+![|602x339](upload://jah9xRmlIOoddRGwE5EnuJ09hdO.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.